[CI] GoogleCloudStorageBlobStoreRepositoryTests.testSnapshotAndRestore failure on 7.8

**Build scan**:
https://gradle-enterprise.elastic.co/s/mfsebuhbp5lls

**Repro line**:

./gradlew ':plugins:repository-gcs:test' --tests "org.elasticsearch.repositories.gcs.GoogleCloudStorageBlobStoreRepositoryTests.testSnapshotAndRestore" \
  -Dtests.seed=F1A04B3D849276C5 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-AU \
  -Dtests.timezone=Europe/Zaporozhye \
  -Druntime.java=11

**Reproduces locally?**:

No

**Applicable branches**:

7.8

**Failure history**:

Found two similar looking failures in May:
https://gradle-enterprise.elastic.co/s/hnatp5gjkc656
https://gradle-enterprise.elastic.co/s/zzz3rgbjsok2c

**Failure excerpt**:

```
java.lang.AssertionError: Only index blobs should remain in repository but found [indices/4FBHGnmIR-esB7D6Jx1iag/0/__l067QhWeRlGEU5saPjHvmg, 
[...] 
indices/oPiiAzvhRL6249ErWUZsMA/4/__6RxT91V7ToKe4-s41wRP0Q]
Expected: a collection with size <0>
     but: collection size was <359>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.elasticsearch.repositories.blobstore.ESMockAPIBasedRepositoryIntegTestCase.tearDownHttpServer(ESMockAPIBasedRepositoryIntegTestCase.java:118)
```
and
```
java.lang.AssertionError: timed out waiting for green state
	at org.junit.Assert.fail(Assert.java:88)
	at org.elasticsearch.test.ESIntegTestCase.ensureColor(ESIntegTestCase.java:941)
```

--- a/test/framework/src/main/java/org/elasticsearch/repositories/blobstore/ESMockAPIBasedRepositoryIntegTestCase.java
+++ b/test/framework/src/main/java/org/elasticsearch/repositories/blobstore/ESMockAPIBasedRepositoryIntegTestCase.java
@@ -26,6 +26,7 @@ import org.apache.http.HttpStatus;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.logging.log4j.message.ParameterizedMessage;
+import org.apache.lucene.util.LuceneTestCase;
 import org.elasticsearch.action.admin.indices.forcemerge.ForceMergeResponse;
 import org.elasticsearch.cluster.metadata.IndexMetadata;
 import org.elasticsearch.common.Strings;
@@ -66,6 +67,8 @@ import static org.hamcrest.Matchers.hasSize;
  * Integration tests for {@link BlobStoreRepository} implementations rely on mock APIs that emulate cloud-based services.
  */
 @SuppressForbidden(reason = "this test uses a HttpServer to emulate a cloud-based storage service")
+// The tests in here do a lot of state updates and other writes to disk and are slowed down too much by WindowsFS
+@LuceneTestCase.SuppressFileSystems(value = {"WindowsFS", "ExtrasFS"})
 public abstract class ESMockAPIBasedRepositoryIntegTestCase extends ESBlobStoreRepositoryIntegTestCase {
 
     /**

